test/simple-proofs.sml

```diff
@@ -88,4 +88,31 @@ struct
         [ Ceq.Eq split [Ceq.Refl, Ceq.Refl]
         , General.Hyp 0
         , Ceq.Step next Ceq.Refl]]);
+
+  (* Pi tests *)
+  val () = mustProve "Pi.FunExt"
+                (PI (UNI 0,
```

Comment:
Yeah, I got about two lines into writing the type by hand and gave up and wrote a parser. :)